src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/PageList.js

```diff
@@ -39,11 +39,12 @@ class PageList extends React.Component<Props> {
     static getDerivedRouteAttributes(route: Route, attributes: AttributeMap) {
         return {
             active: ListStore.getActiveSetting(PAGES_RESOURCE_KEY, getUserSettingsKeyForWebspace(attributes.webspace)),
+            locale: userStore.contentLocale,
```

Comment:
Seems like I didn't state clearly enough what I meant: Since the problem I've found was not based by the missing assignment here, it would have been ok for me to still set this in the `updateRouteAttributesFromUserStoreContentLocale` function. You can leave it here or move it back to the other function, whatever you prefer now, since it makes no difference.